common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -467,6 +467,10 @@ def asset_collection(self):
 MODULESTORE_SETUPS = DIRECT_MODULESTORE_SETUPS + MIXED_MODULESTORE_SETUPS
 MODULESTORE_SHORTNAMES = DIRECT_MS_SETUPS_SHORT + MIXED_MS_SETUPS_SHORT
 SHORT_NAME_MAP = dict(zip(MODULESTORE_SETUPS, MODULESTORE_SHORTNAMES))
+XML_MODULESTORE_MAP = {
+    'XML_MODULESTORE_BUILDER': XmlModulestoreBuilder(),
+    'MIXED_MODULESTORE_BUILDER': MixedModulestoreBuilder([('xml', XmlModulestoreBuilder())])
+}
```

Comment:
Do you expect this to be needed for test modules other than the one that imports it above?  Just wondering why not declare it directly in that module.